core/src/main/java/io/questdb/cairo/AbstractTableNameRegistry.java

```diff
@@ -91,10 +86,10 @@ public void resetMemory() {
         nameStore.resetMemory();
     }
 
-    void setNameMaps(
+    final void setNameMaps(
```

Comment:
What is the point of this method if you made the maps protected?
Please revert this refactoring, including making maps protected. I don't see it adding any value

Reply:
that method is used to swap maps in the read-write implementation

Reply:
addressed in 4231b3bb17c99d32ad3137ba2157ddd3d23c1906